### ISSUE TITLE
Standalone building failed!

### ISSUE THREAD

**Issue by junyussh** [apache/openwhisk#5493]
## Environment details:

I tested the standalone building on Github Codespace.

- OS: Ubuntu 20.04
- Docker version: `Docker version 26.1.4-1, build 5650f9b10226d75e8e9a490a31cc3e5b846e0034`
- Java version: `openjdk 11.0.14.1 2022-02-08 LTS`

## Steps to reproduce the issue:

1.   Open the OpenWhisk repo with Codespace.
2.   Run `./gradlew core:standalone:bootRun`

## Provide the expected results and outputs:

The build command should create a `openwhisk-standalone.jar` file and start the OpenWhisk service on localhost:3232.

## Provide the actual results and outputs:

```
$ ./gradlew core:standalone:bootRun
Build using Scala 2.12

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':core:standalone'.
> Could not resolve all artifacts for configuration ':core:standalone:classpath'.
   > Could not find org.ajoberstar.grgit:grgit-core:3.0.0.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/org/ajoberstar/grgit/grgit-core/3.0.0/grgit-core-3.0.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :core:standalone > com.gorylenko.gradle-git-properties:com.gorylenko.gradle-git-properties.gradle.plugin:2.0.0 > gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.0.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1s

WARNING: Error invoking build scan buildFinished action
        Cannot obtain value from provider of system property 'idea.vendor.name' at configuration time.
Use a provider returned by 'forUseAtConfigurationTime()' instead.
```

## Additional information you deem important:

It says it cannot retrieve the `grgit-core-3.0.0.pom` package due to the url is not exist. I've seen the [standalone test actions](https://github.com/apache/openwhisk/actions/workflows/4-standalone.yaml) on Github. The tests were always passed while the same source code was built failed since yesterday. I suspect the Maven repository changed the package path yesterday resulting in the failure.

**Comment by singhsegv** [comment]
Facing the same issue just now.

**Comment by singhsegv** [comment]
Since the dependency on `grgit-core` is through `com.gorylenko.gradle-git-properties`, updating the build.gradle file for core standalone from `id 'com.gorylenko.gradle-git-properties' version '2.0.0'` to `id 'com.gorylenko.gradle-git-properties' version '2.4.2'` fixed the failing build for me.

**Comment by junyussh** [comment]
> Since the dependency on `grgit-core` is through `com.gorylenko.gradle-git-properties`, updating the build.gradle file for core standalone from `id 'com.gorylenko.gradle-git-properties' version '2.0.0'` to `id 'com.gorylenko.gradle-git-properties' version '2.4.2'` fixed the failing build for me.

Yes, you're right. After fixing the `build.gradle` file, it can build successfully.

**Comment by tess-hartis** [comment]
> Since the dependency on `grgit-core` is through `com.gorylenko.gradle-git-properties`, updating the build.gradle file for core standalone from `id 'com.gorylenko.gradle-git-properties' version '2.0.0'` to `id 'com.gorylenko.gradle-git-properties' version '2.4.2'` fixed the failing build for me.

This also fixed my build issue. Thank you!

**Comment by dgrove-oss** [comment]
A fix was merged to the main branch: #5496.
